build_modules/build.yaml

```diff
@@ -17,7 +17,4 @@ builders:
         - .unlinked.sum
     is_optional: True
     auto_apply: all_packages
-    defaults:
```

Comment:
nit: consider a release default to use the `coarse` strategy? If you do release with dartdevc you would want fewer modules.